front/app/containers/ProjectsShowPage/index.tsx

```diff
@@ -87,12 +95,32 @@ const ProjectsShowPage = memo<Props>(({ project, scrollToEventId }) => {
     ? project.attributes.process_type
     : undefined;
 
+  const smallerThanMinTablet = useBreakpoint('tablet');
+  const { formatMessage } = useIntl();
   const [showModal, setShowModal] = useState<boolean>(false);
   const [phaseIdUrl, setPhaseIdUrl] = useState<string | null>(null);
   const locale = useLocale();
   const appConfig = useAppConfiguration();
   const phases = usePhases(projectId);
 
+  const { events } = useEvents({
+    projectIds: projectId ? [projectId] : undefined,
+    sort: 'newest',
+  });
+
+  const loading = useMemo(() => {
```

Comment:
This is an existing function I just moved up, so that I can use it in my useEffect.